.dart_tool/package_config.json

```diff
@@ -3,235 +3,235 @@
   "packages": [
     {
       "name": "async",
-      "rootUri": "file:///Users/gaurair/.pub-cache/hosted/pub.dev/async-2.11.0",
+      "rootUri": "file:///C:/Users/user/AppData/Local/Pub/Cache/hosted/pub.dev/async-2.11.0",
```

Comment:
add gitignore for these files. You can take a reference from SDK example @Sanketbhavsar19